### PR TITLE
update_template_config

### PR DESCRIPTION
**Pull request by hunght**
Changes
changed the file name of the build
updated metadata

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -85,7 +85,7 @@ const checkExistAndDecryptSigningKey = ({ pw }, signingKeyFile) =>
     const signingKey = JSON.parse(rawSigningKey);
 
     decryptValue(signingKey.private_key_cipher_text, pw)
-      .then(res)
+      .then(singingPrivateKey => res({ singingPrivateKey, signingKey }))
```
Comment (florianlenz) [inline src/utils.js]: why did you change this? For me it makes no sense to resolve with the used singing key.

File: src/webpack.js [hunk 2]
```diff
@@ -47,32 +47,39 @@ const ensureDirectoryExists = filePath => {
...
  * @desc Get meta data by read content file appIcon.png, dappConfig.json
- * @return {void}
+ * @return {object} meta data
  */
 const getDappMetaData = () => {
-  const appIconPath = path.join(process.cwd(), 'appIcon.png');
-  const pathDappConfig = path.join(process.cwd(), 'dappConfig.json');
-  const dappConfig = checkFileExistAndPromptError(pathDappConfig)
-    ? JSON.parse(fs.readFileSync(pathDappConfig, 'utf8'))
+  const pathPackageConfig = path.join(process.cwd(), 'package.json');
```
Comment (florianlenz) [inline src/webpack.js]: you still don't need to construct the absolut path just use `./package.json`

File: src/webpack.js [hunk 3]
```diff
@@ -47,32 +47,39 @@ const ensureDirectoryExists = filePath => {
...
+  const packageConfig = checkFileExistAndPromptError(pathPackageConfig)
+    ? JSON.parse(fs.readFileSync(pathPackageConfig, 'utf8'))
     : {};
+  const dappConfig = packageConfig.pangea_dapp ? packageConfig.pangea_dapp : {};
```
Comment (florianlenz) [inline src/webpack.js]: if there is no config present you can't continue ....

File: src/webpack.js [hunk 4]
```diff
@@ -47,32 +47,39 @@ const ensureDirectoryExists = filePath => {
...
 
-  return dappConfig
-    ? {
-        ...dappConfig,
-        image: checkFileExistAndPromptError(appIconPath)
-          ? base64Encode(appIconPath)
-          : null,
-      }
-    : {};
+  const appIconPath = path.join(process.cwd(), dappConfig.icon_path);
```
Comment (florianlenz) [inline src/webpack.js]: no need to get the full path from my point of view

File: src/webpack.js [hunk 5]
```diff
@@ -47,32 +47,39 @@ const ensureDirectoryExists = filePath => {
...
+  const name = dappConfig.name;
+  if (dappConfig.name && !dappConfig.name['en-us']) {
```
Comment (florianlenz) [inline src/webpack.js]: why don't you use the name variable?

Comment (florianlenz) [reply src/webpack.js]: we use strict check ususally

**Review by florianlenz — CHANGES_REQUESTED** [review]

File: src/webpack.js (hunk 2 above)
Comment (hunght) [reply src/webpack.js]: done

File: src/webpack.js (hunk 3 above)
Comment (hunght) [reply src/webpack.js]: done

File: src/webpack.js (hunk 4 above)
Comment (hunght) [reply src/webpack.js]: done

File: src/webpack.js (hunk 5 above)
Comment (hunght) [reply src/webpack.js]: done

File: src/webpack.js [hunk 6]
```diff
@@ -47,32 +48,43 @@ const ensureDirectoryExists = filePath => {
...
-    : {};
-
-  return dappConfig
-    ? {
-        ...dappConfig,
-        image: checkFileExistAndPromptError(appIconPath)
-          ? base64Encode(appIconPath)
-          : null,
-      }
+  const pathPackageConfig = path.join('./package.json');
```
Comment (florianlenz) [inline src/webpack.js]: do you even need this? I think you have to use `path.join`

File: src/webpack.js [hunk 7]
```diff
@@ -47,32 +48,43 @@ const ensureDirectoryExists = filePath => {
 
 /**
  * @desc Get meta data by read content file appIcon.png, dappConfig.json
```
Comment (florianlenz) [inline src/webpack.js]: `dappConfig.json` we are not using dappConfig anymore

**Review by florianlenz — CHANGES_REQUESTED** [review]

**Comment by florianlenz** [comment]
btw, can you add a few tests?

**Review by florianlenz — APPROVED** [review]
